config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
-
+  # scope :module => 'foreman_gutterball', :path => '/gutterball' do
+  #   resources :content_reports
   scope :module => 'foreman_gutterball', :path => '/gutterball' do
-    resources :content_reports, :only => [:index, :show] do
-      member do
-        get :generate
+    scope :module => 'api' do
+      scope :module =>  'v2' do
+        resources :reports, :only => :index
```

Comment:
i think we'll want to name the resource something other than `reports` because foreman already has some routes with `reports` in them. 

maybe `content_reports` or something similar?